#!/usr/bin/python3


def raise_exception():
    big = "shoes"
    print("{:d}".format(big))
